src/services/lnd/utils-ln-service.ts

```diff
@@ -0,0 +1,44 @@
+import lnService from "ln-service"
```

Comment:
please add these methods to the main lnd service interface

Reply:
thank you for the review, but can you please give more detail about what changes are expected?
Should I add the methods from ln-service or my new methods from this file to which file instead?

Reply:
I mean to add the methods `ILightningService` interface and remove this file

Reply:
done in [242366f](https://github.com/GaloyMoney/galoy/pull/2057/commits/242366f414516e45406203ac6ddfc6552867f80e)